src/file/file.rpc.ts

```diff
@@ -52,15 +52,15 @@ export class RPC {
 
   // Generates a random key for the upload.
   private generateKey(call: any, callback: any) {
-    const key: string = FileService.generateKey();
+    const key:        string = FileService.generateKey();
```

Comment:
This file has an unused import of `IUser` in [line 3](https://github.com/meateam/file-service/blob/a1f222c6d57d630ee113eb08d6e1261698666e7c/src/file/file.rpc.ts#L3).